src/Symfony/Component/Translation/Resources/schemas/xliff-core-1.2-transitional.xsd

```diff
@@ -1838,16 +1842,34 @@ Jan-10-2006
       </xsd:simpleContent>
     </xsd:complexType>
   </xsd:element>
+  <xsd:element name="prop-group">
+    <xsd:complexType>
+      <xsd:sequence maxOccurs="unbounded">
+        <xsd:element ref="xlf:prop"/>
+      </xsd:sequence>
+      <xsd:attribute name="name" type="xsd:string" use="optional"/>
+    </xsd:complexType>
+  </xsd:element>
+  <xsd:element name="prop">
+    <xsd:complexType>
+      <xsd:simpleContent>
+        <xsd:extension base="xsd:string">
+          <xsd:attribute name="prop-type" type="xsd:string" use="required"/>
+          <xsd:attribute ref="xml:lang" use="optional"/>
+        </xsd:extension>
+      </xsd:simpleContent>
+    </xsd:complexType>
+  </xsd:element>
```

Comment:
I had to add declarations `prop-group`/`prop`, otherwise it breaks because we have some validations on XLIFF files.